libs/uw-imap/patches/001-fix_Makefiles_and_shlib.patch

```diff
@@ -0,0 +1,176 @@
+--- a/src/osdep/unix/Makefile	2017-11-13 01:37:58.674011768 +0200
++++ b/src/osdep/unix/Makefile	2017-11-13 03:30:18.204908962 +0200
+@@ -42,7 +42,7 @@
+ # -O3 seems to be safe, but empirical observation from our local expert
+ # indicates that in some (many?) cases -O3 code runs slower than -O2.
+ 
+-GCCOPTLEVEL= -O2
++GCCOPTLEVEL= -Os
```

Comment:
Actually, you shouldn't need to patch this... you can override it via `$(MAKE_FLAGS)`.